extensions/firewall

```diff
@@ -67,14 +67,16 @@ firewalld_up()
 
 	local ZONE=`wicked_config_get_zone "$WICKED_ARGFILE"`
 
-	"$firewalld_cmd" --zone="$ZONE" --change-interface="$WICKED_INTERFACE_NAME" &>/dev/null
+	"$firewalld_cmd" --permanent --zone="$ZONE" --change-interface="$WICKED_INTERFACE_NAME" &>/dev/null
+	"$firewalld_cmd" --reload &>/dev/null
```

Comment:
I think I would go with 
```suggestion
	"$firewalld_cmd" --permanent --zone="$ZONE" --change-interface="$WICKED_INTERFACE_NAME" &>/dev/null
	"$firewalld_cmd" --zone="$ZONE" --change-interface="$WICKED_INTERFACE_NAME" &>/dev/null
```
as recommended, otherwise a `--reload` will loose all other not permanent configurations.

Reply:
Done. I also added a comment why we do that to avoid confusion.